paddle/platform/dynload/dynamic_loader.cc

```diff
@@ -12,13 +12,14 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 
-#include "dynamic_loader.h"
+#include "paddle/platform/dynload/dynamic_loader.h"
```

Comment:
I put this header at the last, but clang-format make it at the first.
So, just let clang-format do it.